pkg/resources/envoy/configmap.go

```diff
@@ -39,12 +39,12 @@ import (
 func (r *Reconciler) configMap(log logr.Logger, envoyConfig *v1beta1.EnvoyConfig) runtime.Object {
 	configMap := &corev1.ConfigMap{
 		ObjectMeta: templates.ObjectMeta(configName(envoyConfig), labelSelector(envoyConfig), r.KafkaCluster),
-		Data:       map[string]string{"envoy.yaml": generateEnvoyConfig(r.KafkaCluster, envoyConfig, log)},
+		Data:       map[string]string{"envoy.yaml": GenerateEnvoyConfig(r.KafkaCluster, envoyConfig, log)},
 	}
 	return configMap
 }
 
-func generateEnvoyConfig(kc *v1beta1.KafkaCluster, envoyConfig *v1beta1.EnvoyConfig, log logr.Logger) string {
+func GenerateEnvoyConfig(kc *v1beta1.KafkaCluster, envoyConfig *v1beta1.EnvoyConfig, log logr.Logger) string {
```

Comment:
all functions seem to be camelCased, don't we want persist that?

Reply:
In Golang, the first letter (capital / lowercase) indicates the visibility: Public or Private [1].
The visibility had to be updated, so that the method could be used in deployment.go

[1] https://ednsquare.com/story/public-and-private-packages-functions-in-golang------KgKYKs

Reply:
Ahh, forgot about this small aspect! Thanks @alungu !